src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php

```diff
@@ -53,7 +54,7 @@ public function enterNode(Node $node, Environment $env): Node
                 $name = new AssignNameExpression($var, $node->getTemplateLine());
                 $this->scope->set('domain', new NameExpression($var, $node->getTemplateLine()));
 
-                return new SetNode(false, new Node([$name]), new Node([$node->getNode('expr')]), $node->getTemplateLine());
+                return new SetNode(false, new Nodes([$name]), new Nodes([$node->getNode('expr')]), $node->getTemplateLine());
```

Comment:
All these usages either need to happen conditionally (when the `Nodes` class exists) or we need to bump the Twig requirement. But I guess dropping Twig 2 support in a patch release is not an option.

Reply:
Done